src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -413,6 +538,14 @@ JSON file.
 
 ### Required fields
 
+{{ MACROS___make_metadata_table(
+   {
+      "RepetitionTime": "REQUIRED",
+      "VolumeTiming": "REQUIRED",
```

Comment:
Wouldn't it make sense to say:

```suggestion
      "VolumeTiming": "REQUIRED, unless `RepetitionTime` and `DelayTime` are defined.",
```

And make similar adjustments for RepetitionTime and DelayTime requirement levels?

Reply:
I notice that there are more fields that could benefit from such an overhaul (AcquisitionDuration)

Reply:
Could we push this one to a separate issue/PR? I think it ties into that weird dependency table for timing-related field and will require more thought than I can put in at the moment.

![image](https://user-images.githubusercontent.com/8228902/124508108-3db7ed00-dd9d-11eb-948c-c001c72d7794.png)

Reply:
ugh, you are right. Let's not open that can of worms (especially not me).